pkg/tfcli/apply.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tfcli
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+
+	cliErrors "github.com/crossplane-contrib/terrajet/pkg/tfcli/errors"
```

Comment:
```suggestion
	clierrors "github.com/crossplane-contrib/terrajet/pkg/tfcli/errors"
```

I'd prefer to avoid camel case package aliases, because camel case is not considered to be a valid package name in Go per https://golang.org/doc/effective_go#package-names:

> the package in src/encoding/base64 is imported as "encoding/base64" but has name base64, not encoding_base64 and not encodingBase64.

See also https://go.dev/blog/package-names which states camel case names are not idiomatic. In this case I would suggest `tferrors` or `clierrors` would be appropriate; for example we frequently import the Kubernetes errors package as `kerrors` to avoid conflicts.